tools/fuzzer/constraints.h

```diff
@@ -107,6 +107,12 @@ class TypeConstraints {
     return retval;
   }
 
+  static TypeConstraints all_non_void() {
+    TypeConstraints retval = all();
```

Comment:
Are there cases when `all()` actually makes sense? All expressions should return something, so we never want `void`, right?

Reply:
That's right. Completely replaced `all()` with `all_non_void()`.